apps/frontend-manage/src/components/questions/QuestionEditModal.tsx

```diff
@@ -140,6 +137,16 @@ function QuestionEditModal({
             })
           }
 
+          case 'KPRIM': {
+            const choicesSchema = baseChoicesSchema
+              .min(4, t('manage.formErrors.NumberQuestionsRequiredKPRIM'))
```

Comment:
I found on StackExchange that cypress might have an option "should('have.length', 4)" to check for an exactly length?

Reply:
thanks for pointing that out. But it seems that `should()` does not work for Yup's ArraySchema anymore. Instead, I used now `length()` to specify the desired length